packages/web-react/src/components/TextArea/README.md

```diff
@@ -5,25 +5,25 @@ and an optional message. It could be disabled or have an error state. The label
-| `message`         | string                        | Validation or help message        |
-| `rows`            | number                        | Number of visible rows            |
-| `validationState` | `success`, `warning`, `error` | Type of validation state          |
-| `isDisabled`      | boolean                       | Whether is field disabled         |
-| `isRequired`      | boolean                       | Whether is field required         |
-| `isLabelHidden`   | boolean                       | Whether is label hidden           |
+| Prop name         | Type                                                | Default | Required | Description                                                                                                               |
+| ----------------- | --------------------------------------------------- | ------- | -------- | ------------------------------------------------------------------------------------------------------------------------- |
+| `id`              | string                                              | -       | yes      | Textarea and label identification                                                                                         |
+| `name`            | string                                              | -       | no       | Textarea name                                                                                                             |
+| `label`           | string                                              | -       | no       | Label text                                                                                                                |
+| `placeholder`     | string                                              | -       | no       | Textarea placeholder                                                                                                      |
+| `value`           | string                                              | -       | no       | Textarea value                                                                                                            |
+| `maxLength`       | number                                              | -       | no       | Maximum number of characters                                                                                              |
+| `message`         | string                                              | -       | no       | Validation or help message                                                                                                |
+| `rows`            | number                                              | -       | no       | Number of visible rows                                                                                                    |
+| `validationState` | `success`, `warning`, `danger`, `error(deprecated)` | -       | no       | Type of validation state. See [dictionaries]. The value "error" in the dictionary will be replaced by the value "danger". |
```

Comment:
```suggestion
| `validationState` | `success`, `warning`, `danger`, `error` (deprecated) | -       | no       | Type of validation state. See [dictionaries]. The value "error" in the dictionary will be replaced by the value "danger". |
```

Reply:
fixed & done

Reply:
Linking it do deprecation info sounds fine. Would you do that on all readmes you've just modified, please? Thanks :-)

Reply:
done